lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -1784,45 +1813,35 @@ class ByteCodeBufferBuilder
         return sizeof(serialization_alignment TStructType);
     }
 
-    uint32 AddPropertyIdOfFormals(BufferBuilderList & builder, FunctionBody * function)
+    uint32 AddPropertyIdOfFormals(BufferBuilderList & builder, PropertyIdArray * propIds, FunctionBody * function)
```

Comment:
>propIds [](start = 81, length = 7)

`Assert(propIds != nullptr)` ?

Reply:
Doesn't feel like the assert serves much value.....